proto/axelarnet/v1beta1/service.proto

```diff
@@ -66,3 +67,12 @@ service MsgService {
     };
   }
 }
+
+// QueryService defines the gRPC querier service.
+service QueryService {
+  rpc PendingIBCTransferCount(PendingIBCTransferCountRequest)
+      returns (PendingIBCTransferCountResponse) {
+    option (google.api.http).get =
+        "/axelar/axelarnet/v1beta1/ibc-transfer-count";
```

Comment:
```suggestion
        "/axelar/axelarnet/v1beta1/ibc_transfer_count";
```
Cosmos SDK endpoints use `_`: https://axelar-lcd.quickapi.com/swagger/
I'll fix the existing queries in my enabling swagger PR

Reply:
gotcha